flox-bash/lib/commands/activate.sh

```diff
@@ -73,7 +74,8 @@ _usage["activate"]="activate environment:
 
 function floxActivate() {
 	trace "$@"
-	local -a environments=($1); shift
+	local _target_environment="$1";
+	local -a environments=("$1"); shift
```

Comment:
Will quoting this mess up the splitting?

Reply:
No `"$1"` is already split. Should always quote `argv` expansion as either:
```
set -- a b c;
$ printf '%s\n' "$1 $2 $3";
a b c
$ printf '%s\n' "$*";
a b c
$ printf '%s\n' "$@";
a
b
c
$ printf '%s\n' "$1" "$2" "$3";
a
b
c
```

Reply:
You're right the env list is quoted passed into `argv[1]`. I've changed it to use `read -ra` and am testing now.
Good catch. We should add a test case that uses multiple envs on the CLI.

Reply:
fixed it.